src/orange/orangeinp/IntersectRegion.hh

```diff
@@ -138,6 +138,56 @@ class Cone final : public IntersectRegionInterface
     real_type hh_;
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * A *z*-aligned cylinder centered on the origin, with top/bottom cuts.
+ *
+ * The shape is defined with a radius, half-height, and the outward-facing
+ * normals of the cutting planes, passing through (0, 0, +/-hh). This class
+ * does not support the case where both cut planes are perpendicular to the z
+ * axis; a Cylinder should be used in this case.
```

Comment:
Could you use `\f$(0, 0, \pm  \mathrm{hh})\f$` , italicize `z`, and add this to `doc/implementation/geometry/orange.rst`?